gandalf.js

```diff
@@ -1,26 +1,28 @@
 /**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.